.script/tests/KqlvalidationsTests/CustomTables/darktrace_model_alerts_CL.json

```diff
@@ -89,6 +89,10 @@
         "Name": "url_s",
         "Type": "String"
       },
+      {
+        "Name": "breachUrl_s",
+        "Type": "String"
+      },
       {
```

Comment:
@JustasZDT, please add "triggeredComponents_s" in custom table as the kql validations are failing.

Reply:
@v-sabiraj triggeredComponents_s has been added